shell/app/modules/msp/monitor/browser-insight/common/components/sort-list/sub-tab.scss

```diff
@@ -0,0 +1,3 @@
+.sub-tab {
+  margin-bottom: 20px;
```

Comment:
Why not tailwind?

Reply:
the classname `sub-tab` is used in some other place. need the classname